### DISCUSSION TITLE
Sentence capitalization and paragraph indent

### DISCUSSION THREAD

**Discussion by JPhB** [vkbo/novelWriter#1357] [Ideas]
I'd add capitalization of the first letter of a sentence, and also a paragraph indentation option. It would make the text easier to read.

**Comment by vkbo** [comment]
I've considered adding automatic capitalisation as an optional feature, so feel free to open a feature issue on that and I'll queue it up for the editor updates.

For indentation, that will not happen for the editor at least. The editor is plain text, so it follows the standard plain text approach with blank lines between paragraphs. For manuscripts, indented paragraphs are coming. It's suggested in #906 and planned for inclusion in the next minor release, 2.1 (see #1269).
